pkg/test/testPackages.proj

```diff
@@ -173,13 +173,19 @@
   <Target Name="RestoreProjects"
           DependsOnTargets="GenerateProjects">
     <Message Importance="High" Text="*** Restoring ***" />
-    <Exec Command="$(TestRestoreCommand) &quot;$(TestProject)&quot;" EnvironmentVariables="@(CliEnvironment)" ContinueOnError="ErrorAndContinue" StandardOutputImportance="High" />
+    <Exec Command="$(TestRestoreCommand) &quot;$(TestProject)&quot;" EnvironmentVariables="@(CliEnvironment)" ContinueOnError="ErrorAndContinue" StandardOutputImportance="High">
```

Comment:
Looks like we set ErrorAndContinue explicitly to allow all the restores to happen perhaps we should capture this exit code and error somewhere later if there was a failure.

Reply:
I think we do it just so that we can see all restore issues at once, we do the same for the actual build of the projects. I think it is probably enough if we error after the restore, since if there where errors with restore, build most likely will have way more errors that might not make much sense. That said, I could add a target after restore and build that will basically error if either restore or build failed if you want me to.

Reply:
unless I'm missing something what you have done would be equivalent to removing the ContinueOnError setting. Will that allow all the other projects to be tested?

Reply:
Oh I thought we were calling restore to each individual project, but I see now that we are calling it to the root project which is restoring all individual projects. @ericstj do you think it adds any value to go to the build step of the tests when restore failed for some?

Reply:
I think its OK to halt if restore fails.